UsabillaSampleApp/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "react": "16.13.1",
     "react-native": "0.64.1",
-    "usabilla-react-native": "^0.12.0"
+    "usabilla-react-native": "file:../"
```

Comment:
To try the latest changes in Sample project, switched this to use relative project.